R/init.R

```diff
@@ -385,6 +395,54 @@ if (interactive() &&
         page_viewer = page_viewer
       )
 
+      #vsc-r-api
+      RESPONSE_TIMEOUT <- 30
+      response_lock_file <- file.path(dir_extension, "response.lock")
+      response_file <- file.path(dir_extension, "response.log")
+      addin_registry <- file.path(dir_extension, "addins.json")
```

Comment:
I guess we should write `addins.json` to session temp folder? Then we could allow different sessions to have different set of addins, which might be useful if the sessions are started with independent user libraries (e.g. `renv` is used). The command shows the addins of the attached session.

Reply:
yep good idea